devel/addon/certmanager/install.sh

```diff
@@ -24,6 +24,7 @@ NAMESPACE="${NAMESPACE:-cert-manager}"
 RELEASE_NAME="${RELEASE_NAME:-cert-manager}"
 # Default feature gates to enable
 FEATURE_GATES="${FEATURE_GATES:-ExperimentalCertificateSigningRequestControllers=true,ExperimentalGatewayAPISupport=true,AdditionalCertificateOutputFormats=true}"
+FEATURE_GATES_CA_INJECTOR="${FEATURE_GATES_CA_INJECTOR:-ServerSideApply=true}"
```

Comment:
I don't think we want to run this by default - shouldn't this be automatically toggled on by our AllAlpha=true flag passed down from CI config? (so we only run alpha tests when alpha jobs are enabled)